packages/rest/src/controllers/proofs/ProofController.ts

```diff
@@ -152,46 +228,76 @@ export class ProofController extends Controller {
     }
   }
 
-  /**
-   * Creates a presentation request not bound to any proposal or existing connection
-   *
-   * @param request
-   * @returns ProofRequestMessageResponse
-   */
-  @Post('/request-outofband-proof')
-  @Example<{ proofUrl: string; proofRecord: ProofRecordProps }>({
-    proofUrl: 'https://example.com/proof-url',
-    proofRecord: ProofRecordExample,
-  })
-  public async requestProofOutOfBand(@Body() request: Omit<RequestProofOptions, 'connectionId'>) {
-    const { proofRequestOptions, ...requestOptions } = request
-    const proof = await this.agent.proofs.createOutOfBandRequest(proofRequestOptions, requestOptions)
-
-    return {
-      proofUrl: `${this.agent.config.endpoints[0]}/?d_m=${JsonEncoder.toBase64URL(
-        proof.requestMessage.toJSON({ useLegacyDidSovPrefix: this.agent.config.useLegacyDidSovPrefix })
-      )}`,
-      proofRecord: proof.proofRecord,
-    }
-  }
+  // TODO: Represent out-of-band proof
```

Comment:
This can be just `create-request` and then return an object with the proofRecord and the requestMessage (follow API from `createRequest` on proofs module. Then the user can call the oob api with the message